### PR TITLE
Fix for issue #276

### PR DESCRIPTION
**Pull request by tyler46**
I think now it explicit that rabbitmq is needed in order to run nameko example service.

### CODE REVIEW

**Comment by davidszotten** [comment]
Hi,

Thanks for the contribution!

A small note: You write "to run nameko you need [rabbitmq]" which isn't quite right. You need rabbitmq to use the built-in rpc functionality that's used in the example, but it's not required by the core framework. I'm not sure what the bast way to phrase things but i think it's worth making this distinction clear

(@mattbennett This is making me wonder if we should look for a different hello-world example. http has fewer dependencies but i think may be less good at making it clear how we're different from web frameworks like flask)

**Comment by tyler46** [comment]
Hi @davidszotten,

I thought by writing "nameko has built its RPC and other features over AMQP", I was explaining why we  need [rabbimtq]. Of course I can rephrase it to be more accurate. Do you think something like the following would be better:

> Before running nameko service, please be sure that you have a [rabbitmq] instance running, since nameko built-in rpc functionality relies on AQMP that is being implemented by [rabbitmq].

I agree with you that using a hello-world example using http entrypoint would be less good that current one.

**Comment by mattbennett** [comment]
@tyler46 thanks for the PR.

The difficulty here is that _this specific_ example requires RabbitMQ, because it's using the built-in AMQP RPC features. I think it'd be sufficient to just say that.

On the project page we feature an RPC example and an HTTP example, but it may be distracting to have both here. The RPC one has the advantage of also being able to show the shell usage underneath.

**Comment by tyler46** [comment]
Hi @mattbennett,

OK I will update PR accordingly.

Thank you
